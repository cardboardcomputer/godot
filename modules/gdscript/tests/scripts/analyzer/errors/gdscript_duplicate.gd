const TestClass = preload("gdscript_duplicate_class.notest.gd")

func test():
	# (TestClass as GDScript).duplicate() exists
	TestClass.duplicate()
