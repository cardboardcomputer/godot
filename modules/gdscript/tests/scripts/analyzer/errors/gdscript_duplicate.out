GDTEST_ANALYZER_ERROR
Cannot call non-static function "duplicate()" on a class directly. Make an instance instead.
